src/main/java/edu/greenblitz/robotName/commands/arm/MoveArmToIntakePosition.java

```diff
@@ -0,0 +1,14 @@
+package edu.greenblitz.robotName.commands.arm;
+
+import edu.greenblitz.robotName.subsystems.arm.elbow.ElbowConstants;
+import edu.greenblitz.robotName.subsystems.arm.wrist.WristConstants;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+
+public class MoveArmToIntakePosition extends ParallelCommandGroup {
+
```

Comment:
I would have said something meaningful if I could understand any of this code

Reply:
thete is no need to be hurtful